src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchTrackingService.java

```diff
@@ -276,7 +276,7 @@ private SpeakerSwitchView buildSwitch(IOFSwitch sw) {
         return new SpeakerSwitchView(new SwitchId(sw.getId().getLong()),
                                      (InetSocketAddress) sw.getInetAddress(),
                                      (InetSocketAddress) (sw.getConnectionByCategory(
-                                                     LogicalOFMessageCategory.MAIN).getRemoteInetAddress()),
+                                                     LogicalOFMessageCategory.MAIN).getLocalInetAddress()),
```

Comment:
Because it was incorrect - it grabs switch socket address while it was supposed to grab speaker socket address. This field wasn't used until now, so error was not noticed.